systests/microprofile/client/weld/pom.xml

```diff
@@ -85,17 +92,65 @@
             <artifactId>johnzon-jsonb</artifactId>
```

Comment:
Could you please change this one to:

```
<dependency>
            <groupId>org.apache.johnzon</groupId>
            <artifactId>johnzon-jsonb</artifactId>
            <classifier>jakarta</classifier>
            <exclusions>
                <exclusion>
                    <groupId>*</groupId>
                    <artifactId>*</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
        <dependency>
            <groupId>org.apache.johnzon</groupId>
            <artifactId>johnzon-mapper</artifactId>
            <classifier>jakarta</classifier>
        </dependency>
```

(the same as other POM), thanks

Reply:
Done.